### PR TITLE
docs($templateCache): highlight the $templateCache service

### PR DESCRIPTION
**Pull request by cironunes**

### CODE REVIEW

File: src/ng/cacheFactory.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ function $CacheFactoryProvider() {
  * the document, but it must be a descendent of the {@link ng.$rootElement $rootElement} (IE,
  * element with ng-app attribute), otherwise the template will be ignored.
  *
- * Adding via the $templateCache service:
+ * Adding via the ```$templateCache``` service:
```
Comment (gkalpak) [inline src/ng/cacheFactory.js]: Why so many backticks ? (Do you really like them or is there another reason ?)

Comment (cironunes) [inline src/ng/cacheFactory.js]: I didn't know that it's possible to use single backticks to format as code in markdown. :smile:

**Comment by cironunes** [comment]
Closed here and opened #11294

**Comment by gkalpak** [comment]
For the record, you didn't have to open a new PR. You could update your branch and force-push.

**Pull request closed without merging**